plugin/schema.go

```diff
@@ -50,13 +50,9 @@ func schema(e Entry, includeChildren bool) *EntrySchema {
 	// TODO: Handle external plugin schemas
 	switch e.(type) {
 	case *externalPluginRoot:
-		return &EntrySchema{
-			TypeID: "external-plugin-root",
-		}
+		return nil
 	case *externalPluginEntry:
```

Comment:
I think `schema` should be optional for external plugins to make things easier for Bash plugin authors. That's also why `/fs/schema` can return `null`.